client/packages/host/src/components/Initialise/InitialiseLayout.tsx

```diff
@@ -50,6 +52,7 @@ export const InitialiseLayout = ({
         flexDirection="column"
       >
         <Box>
+          {SiteInfo}
```

Comment:
Random placement really, not sure where it should go, but realised `alt` key wasn't going to work for Android so a way to go back to discovery is required (both in iniitalisation and login)